bin/oscapd-evaluate

```diff
@@ -307,10 +307,13 @@ def cli_scan(args, config):
             )
             json_data["Profile"] = args.profile
 
-            with io.open(os.path.join(
-                    full_output_dir, "std.xml"), "w",
-                    encoding="utf-8") as f:
+            arf_filepath = os.path.join(full_output_dir, "std.xml")
```

Comment:
@mpreisler Yes, we could. That would be better name. I didn't change it because it wasn't necessary for the new feature.